Detections/AWSCloudTrail/AWS_LoadBalancerSecGroupChange.yaml

```diff
@@ -1,9 +1,10 @@
 ﻿id: c7bfadd4-34a6-4fa5-82f8-3691a32261e8
 name: Changes to AWS Elastic Load Balancer security groups
 description: |
-  'Monitoring Changes to Elastic Load Balancer specific security groups.
-  More information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255 
-  and https://aws.amazon.com/elasticloadbalancing/.'
+  'Elastic load balancers distributes incoming traffic across multiple instances in multiple availability Zones. This increases the fault tolerance of your applications. 
```

Comment:
Elastic Load Balancers (keep it consistent)

Reply:
Done. Thanks